[PATCH] disable tests

The tests need C++ support and add to build time, so disable them for BR.

Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
---
 Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: evemu-1.2.0/Makefile.am
===================================================================
--- evemu-1.2.0.orig/Makefile.am
+++ evemu-1.2.0/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = src tools python test
+SUBDIRS = src tools python
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = evemu.pc
